src/ploomber_cli/cli.py

```diff
@@ -13,6 +14,22 @@
 VERSION_KWARGS = dict(package_name="ploomber") if CLICK_VERSION >= 8 else dict()
 
 
+def cloud_deprecation_warning():
+    """A resuable function to pass a warning to all cloud_ functions
+
+
+    Return: None
+    """
```

Comment:
no need to add a docstring here

Reply:
The lint fails otherwise. Should I remove it anyway?

Reply:
I don't think the lint requires docstrings. what error did you get?

Reply:
Seems to have passed this time. Squashing and pushing in a moment.